lumigator/python/mzai/backend/backend/tests/integration/api/routes/test_api_workflows.py

```diff
@@ -18,7 +18,11 @@
 )
 
 from backend.main import app
-from backend.tests.conftest import TEST_CAUSAL_MODEL, TEST_INFER_MODEL, TEST_SUMMARY_MODEL
+from backend.tests.conftest import (
+    TEST_CAUSAL_MODEL,
+    TEST_INFER_MODEL,
+    TEST_SUMMARY_MODEL,
```

Comment:
i don't think summary and infer model actually get used, can remove

Reply:
I'd like to keep them in the conftest for reference, although I'll remove them from the imports here, yes. Is that ok?